src/event-replay/parquet-based/post-replay-db.ts

```diff
@@ -0,0 +1,17 @@
+import { logger } from '../../logger';
+import { PgWriteStore } from '../../datastore/pg-write-store';
+
+(async () => {
+  const db = await PgWriteStore.connect({
+    usageName: 'post-event-replay',
+    skipMigrations: true,
+    withNotifier: false,
+    isEventReplay: true,
+  });
+
+  // Refreshing materialized views
+  logger.info({ component: 'event-replay' }, `Refreshing materialized views`);
```

Comment:
Should this also be re-enabling indexes? e.g. `db.toggleTableIndexes(...)`

Reply:
Yep. It's missing re-enable the indexes. Fixed.